egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -314,30 +314,49 @@ def train(args, run_opts, background_process_handler):
     default_egs_dir = '{0}/egs'.format(args.dir)
     if (args.stage <= -3) and args.egs_dir is None:
         logger.info("Generating egs")
+        # If num of egs dirs in args.egs_dir > 1,
+        # it is correspond to multilingual training and 
+        # it should generate egs dir for each languages.
+        # The last dir corresponds to multilingual egs directory,
+        # that is generated using this script, but 
+        # it requires single egs dirs to exist.
+        multi_egs_dir = args.egs_dir.split()
```

Comment:
It seems I already generate egs outside train_raw.py for multilingual recipe local/nnet3/run_tdnn_multilingual.sh (I should revert these changes!) and pass it ad egs-dir to train.py.